src/app/client/src/app/modules/workspace/components/content-editors/new-collection-editor/new-collection-editor.component.ts

```diff
@@ -324,7 +324,12 @@ export class NewCollectionEditorComponent implements OnInit, OnDestroy {
           isRootOrgAdmin: this.userService.userProfile.rootOrgAdmin
         },
         channelData: this.frameworkService['_channelData'],
-        cloudStorageUrls : this.userService.cloudStorageUrls
+        cloudStorageUrls : this.userService.cloudStorageUrls,
+        cloudStorage: {
+          presigned_headers: {
+             'x-ms-blob-type': 'BlockBlob'
```

Comment:
can you put this as a variable and not as a string as this can change if needed

Reply:
@rajeevsathish Updated as suggested. Please review it.